dir.props

```diff
@@ -546,6 +498,7 @@
     <SkipTests Condition="'$(SkipTests)'=='' and ('$(OsEnvironment)'=='Windows_NT' and '$(TargetsWindows)'!='true' and '$(OSGroup)'!='AnyOS')">true</SkipTests>
   </PropertyGroup>
 
-  <Import Project="$(RoslynPropsFile)" Condition="'$(OsEnvironment)'=='Unix' and Exists('$(RoslynPropsFile)')" />
-
+  <!-- Use Roslyn Compilers to build -->
+  <Import Project="$(RoslynPropsFile)" Condition="'$(OSEnvironment)'!='Unix' and Exists('$(RoslynPropsFile)') and '$(UseRoslynCompilers)'!='false'" />
```

Comment:
I wanted to combine these because I wanted to align these on Windows / Unix (in fact I did with a previous commit).  I then backtracked a bit because I didn't feel I had clear context on the reason this was split in the first place, so I reverted that change to this.

I also considered moving this to buildtools, but I didn't want to force other repo's that upgrade to this version of buildtools to consume the Roslyn compiler toolset update to deal with duplicate imports, especially because it's not clear to me if other repo's are following the same pattern here or doing something different.  

To both of your points, I agree that it would be cleaner.  I'm ok with un-splitting the imports with this change, but i'll need to file an issue to move the imports to buildtools as a separate task.